features/admin.console-settings.v1/pages/console-settings-page.tsx

```diff
@@ -79,14 +81,16 @@ const ConsoleSettingsPage: FunctionComponent<ConsoleSettingsPageInterface> = (
                         InputProps={ {
                             endAdornment: (
                                 <Tooltip
-                                    title="Copy"
+                                    title={ isCopied ? "Copied!" : "Copy" }
```

Comment:
Let's use i18n strings instead of english literals.

Reply:
translation files in `modules/i18n` should be compiled so that final translation JSON bundles are generated that can be used by `react-i18next` package in console app. You can just run `pnpm build` from the project root and all changed packages will be compiled. After that, start the console in development mode.

Also make sure to replace the string literals with the i18n function calls. You can find some examples by searching the usage of `useTranslation` react hook.

Reply:
or is there any website which you can refer for translation